src/backend/joanie/settings.py

```diff
@@ -473,6 +473,31 @@ class Base(Configuration):
     # Celery
     CELERY_BROKER_URL = values.Value("redis://redis:6379/0")
     CELERY_BROKER_TRANSPORT_OPTIONS = values.DictValue({})
+    CELERY_RESULT_BACKEND = values.Value("redis://redis:6379/0")
```

Comment:
This is not usable for us in production as we are using sentinel. An other way to use it is to set it to `""` here and define it in the environment variable.

Reply:
Shouldn't we have the same problem with CELERY_BROKER_URL, then ?

Reply:
No because the `CELERY_BROKER_TRANSPORT_OPTIONS` is used to configure sentinel and unfortunately this settings does not exists for the result backend

Reply:
OK, I see.
But maybe we can just remove it, as is it only needed with flower.
We can also remove flower install, btw.

Reply:
If flower is not used anymore, then yes you can remove it.